docker/local-universe/Makefile

```diff
@@ -14,6 +14,12 @@ clean:
 	rm -rf certs &&							\
 	rm -f local-universe.tar.gz || 0
 
+static:
+	docker build -t universe-static -f Dockerfile.static .
+
+static-base:
+	docker build -t universe-base -f Dockerfile.static.base .
```

Comment:
Shouldn't this depend on the `clean` and `certs` targets, like `base`, and also the `static` target, since it needs the `universe-static` image?